src/PKSim.Infrastructure/Services/PresenterLogger.cs

```diff
@@ -13,20 +13,15 @@ public class PresenterLogger : ILogger
    {
       private readonly IEventPublisher _eventPublisher;
       public string Name { get; }
-      private readonly LogLevel _logLevel;
 
-      public PresenterLogger(string name, LogLevel logLevel)
+      public PresenterLogger(string name)
       {
          _eventPublisher = IoC.Resolve<IEventPublisher>();
-         _logLevel = logLevel;
```

Comment:
Why was it removed?

Reply:
@msevestre @georgeDaskalakis We do not need to keep track of this anymore. By setting the minimum log level with builder.SetMinimumLevel when configuring the LoggingBuilder we ensure the log method will not be called at all. We could still use the logLevel for filtering when in combination to something more but that is not the case now so we remove it for now and the configuration step will also be smoother.